### DISCUSSION TITLE
How works enable_ws_compression ?

### DISCUSSION THREAD

**Discussion by serhiihiba** [anycable/anycable#206] [Q&A]
How `--enable_ws_compression=true` works? How can i know my WS message is compressed/decompressed and what data type should i send to server to know it will be auto decompressed (gzip/brotli idk)?😬 So many questions, really not clear about that, would be fine to see some JS example😅

**Comment by palkan** [comment]
The flag allows WebSocket clients to communicate their compression preferences with the server. It's all done at the WebSocket protocol level, we have little control over it. 

I've read about WS compression here: https://anycable.substack.com/p/any-cables-monthly-5-november-2022 . There is a bunch of links on the topic explaining WS compression in depth.
